README.md

```diff
@@ -164,28 +164,35 @@ KubeClarity includes a CLI that can be run locally and especially useful for CI/
 It allows to analyze images and directories to generate SBOM, and scan it for vulnerabilities.
 The results can be exported to KubeClarity backend.
 
-### Compilation
```

Comment:
I think that we should keep this as one of the options (last), with a command to add it to path under kubeclarity-cli